### PR TITLE
Add method to modify drops of BlockMocks

### PR DESCRIPTION
**Pull request by XDMAN500**
# Description

Add method to modify drops of BlockMocks

# Checklist
The following items should be checked before the pull request can be merged.
- [X] Code follows existing style.
- [X] Unit tests added (if applicable).

### CODE REVIEW

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java [hunk 1]
```diff
@@ -643,8 +642,7 @@ public VoxelShape getCollisionShape()
 	@Override
 	public @NotNull Collection<ItemStack> getDrops(@NotNull ItemStack tool, Entity entity)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return drops;
```
Comment (Thorinwasher) [inline src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Probably a good idea to return a copy of the drops collection

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java [hunk 2]
```diff
@@ -695,4 +693,9 @@ public boolean canPlace(@NotNull BlockData data)
 		throw new UnimplementedOperationException();
 	}
 
+	public void setDrops(Collection<ItemStack> items)
+	{
+		drops = items;
```
Comment (Thorinwasher) [inline src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Probably a good idea to copy this to a list

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java [hunk 3]
```diff
@@ -607,8 +606,8 @@ public void setBlockData(@NotNull BlockData data)
 	@Override
 	public void setBlockData(@NotNull BlockData data, boolean applyPhysics)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.material = data.getMaterial();
+		this.blockData = data;
```
Comment (Thorinwasher) [inline src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Copy the data

**Review by Thorinwasher — COMMENTED** [review]
Just a minor comment to deal with weird behavior from mutability

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java (hunk 1 above)
Comment (XDMAN500) [reply src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Copied.

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java (hunk 2 above)
Comment (XDMAN500) [reply src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Copied.

File: src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java (hunk 3 above)
Comment (XDMAN500) [reply src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java]: Copied.

**Review by Thorinwasher — APPROVED** [review]

**Pull request merged**